beacon_node/http_api/src/lib.rs

```diff
@@ -4604,6 +4604,36 @@ pub fn serve<T: BeaconChainTypes>(
             },
         );
 
+    // POST lighthouse/custody/backfill
+    let post_lighthouse_custody_backfill = warp::path("lighthouse")
+        .and(warp::path("custody"))
+        .and(warp::path("backfill"))
+        .and(warp::path::end())
+        .and(task_spawner_filter.clone())
+        .and(chain_filter.clone())
+        .then(
+            |task_spawner: TaskSpawner<T::EthSpec>, chain: Arc<BeaconChain<T>>| {
+                task_spawner.blocking_json_task(Priority::P0, move || {
```

Comment:
I don't think this matters if user's using this correctly and if this function is fast - in terms of priority this isn't higher priority than getting node in sync or performing validator duties - i think it may be worth downgrading? in the unlikely case that the `cached_head_read_lock` was held elsewhere, we could delay higher priority tasks.